Updates to Personalized Variant for GA (#4070)
braze-inc/braze-docs: _docs/_user_guide/engagement_tools/testing/multivariant_testing/multivariate_analytics.md

#### How personalized variants are selected

An individual user's recommended message is the sum of the effects of their specific recency, frequency, and tenure. Recency, frequency, and tenure are split into buckets, as illustrated in the **User Characteristics** table. The time range of each bucket is determined by the data for users in each individual campaign and will change from campaign to campaign. 

Each bucket can have a different contribution or "push" towards each message variant. The strength of the push for each bucket is determined from user responses in the initial send using logistic regression. This table only summarizes the results by displaying which variant users in each bucket tended to engage with. But, any individual user's actual personalized variant depends on the sum of the effects of the three buckets they're in—one for each characteristic.

{% endtab %}
{% tab Personalized Variant %}

The **Personalized Variant** tab shows the results of the second send, where each remaining user was sent the variant they were most likely to engage with.

> The three cards on this page show your projected lift, overall results, and the projected results if you sent just the winning variant instead. Even if Personalized Variant fails to provide any lift, which can sometimes happen, the result is equivalent to sending only the winning variant, i.e. a traditional A/B test. 

Reviewer: ```suggestion
The three cards on this page show your projected lift, overall results, and the projected results if you sent just the winning variant instead. Even if there's no lift, which can sometimes happen, the result is the same as sending only the winning variant (a traditional A/B test). 
```

#### How personalized variants are selected

> An individual user's recommended message is the sum of the effects of their specific recency, frequency, and tenure. Recency, frequency, and tenure are split into buckets, as illustrated in the **User Characteristics** table. The extent or width of each bucket is determined by the data for users in each individual campaign and will therefore change from campaign to campaign. 

Author: It won't always be days unfortunately. Could be hours

> An individual user's recommended message is the sum of the effects of their specific recency, frequency, and tenure. Recency, frequency, and tenure are split into buckets, as illustrated in the **User Characteristics** table. The extent or width of each bucket is determined by the data for users in each individual campaign and will therefore change from campaign to campaign. 

Reviewer: Hmm okay. Time range maybe?

> Each bucket can have a different contribution at a different "strength" for each message variant. This table only summarizes which variant users in each bucket tends to engage with. It does not guarantee that the "strength" of any one bucket will dominate the others. Therefore a user's personalized variant depends on the sum of the effects of the three buckets they're in—one for each characteristic.

Reviewer: ```suggestion
Each bucket can have a different contribution at a different "strength" for each message variant. This table only summarizes which variant users in each bucket tend to engage with. It does not guarantee that the "strength" of any one bucket will dominate the others. Therefore a user's personalized variant depends on the sum of the effects of the three buckets they're in—one for each characteristic.
```
